rest/bulk_api.go

```diff
@@ -285,12 +287,12 @@ func (h *handler) handleRepair() error {
 
 	repairBucketResult, repairDocsErr := repairBucket.RepairBucket()
 	if repairDocsErr != nil {
-		return fmt.Errorf("Error repairing bucket: %v", err)
+		return pkgerrors.Wrapf(err, "Error repairing bucket")
```

Comment:
RepairBucket() is internal - I don't know if we need to wrap here?

Reply:
I pushed this down.  It should never happen, and if it does I think it's useful to have a way to get at the stacktrace

Reply:
Pushed it even lower down to gocb boundary